final_prep/README.md

```diff
@@ -45,30 +45,36 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+![8EB54962-504C-4E1F-A193-D64DFCD49038](https://user-images.githubusercontent.com/102932532/164861719-40acf9fc-fee7-4fa3-9f30-23201e896430.jpeg)
+
 - `Add Week 2 Screenshot Here`
+![9F1159E8-DF49-439E-A0C4-594CA6E03AB0](https://user-images.githubusercontent.com/102932532/164861725-6cfc3a89-c235-4df0-9232-aa9bf85be77c.jpeg)
```

Comment:
This looks like a realistic week at Turing hours-wise - do you really think you'll be working Fri and Sat evenings??